### PR TITLE
Hotfix/force tls use on client

### PR DESCRIPTION
**Pull request by leonardo-barbosa**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -45,6 +45,8 @@
     </repository>
   </distributionManagement>
   <properties>
+    <maven.compiler.source>1.6</maven.compiler.source>
```
Comment (murilohns) [inline pom.xml]: o que essa linha faz?

Comment (leonardo-barbosa) [reply pom.xml]: Ela muda a versão do `maven compiler` para a 1.6, visto que a padrão era a 1.5 e não funciona no JDK 9. @murilohns

**Comment by leonardo-barbosa** [comment]
@jonatasmaxi I got your point, but I'm not entirely sure yet.

@henriquekano can you please review again? I made some changes since looks like JDK 6 isn't compatible with TLS 1.2, also I would like to see you opinion about Jonatas opinion above.

**Comment by henriquekano** [comment]
@leonardo-barbosa @jonatasmaxi since we're ending support for java 5, bumping the major version sounds good. But since it's a upgrade that might break existing integrations, it'd be good to at least investigate who still uses that version and warn them before making the upgrade.
BTW, java 5 isn't supported officially for quite some time, as far as I googled. Since 2009.

**Pull request merged**